bee-network/bee-autopeering/src/proto/packet.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
```

Comment:
All proto files are commented except this one

Reply:
Basically the same explanation as above: https://github.com/iotaledger/hive.go/blob/master/autopeering/server/proto/packet.proto

I want to keep the "interface" (which those proto files represent) identical between Bee and Hornet.